internal/db/sqlite/init/00-indexes.sql

```diff
@@ -0,0 +1,27 @@
+PRAGMA journal_mode=WAL
```

Comment:
Adding a comment here as it's the first `PRAGMA` I encountered:

Consider which autovacuum mode to use or handle it at the application level:

https://www.sqlite.org/pragma.html#pragma_auto_vacuum

Also run `PRAGMA optimize` while opening a DB connection:

https://www.sqlite.org/pragma.html#pragma_optimize